soc/nxp/imxrt/Kconfig.soc

```
@@ -0,0 +1,10 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_IMXRT
+	bool
+
+config SOC_FAMILY
+	default "imxrt" if SOC_FAMILY_IMXRT
```

not an issue but other PRs are doing `nxp_<series>` so just seems out of place here. If you did want to change to the nxp prefix:
```suggestion
config SOC_FAMILY_NXP_IMXRT
	bool

config SOC_FAMILY
	default "nxp_imxrt" if SOC_FAMILY_NXP_IMXRT
```

---

have seen below that the family name here is actually wrong and does have nxp prefix so please update as shown